frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/tests/common.unit.spec.tsx

```diff
@@ -439,14 +439,14 @@ describe("Static Embed Setup phase", () => {
           activeTab: "Look and Feel",
         });
 
-        await userEvent.click(screen.getByText("Transparent"));
+        await userEvent.click(screen.getByText("Dark"));
```

Comment:
since `transparent` is not a theme any more we can just test with `dark` theme instead.